DesktopModules/Vanjaro/Core/Library/PackageList.txt

```diff
@@ -4,6 +4,7 @@ themes-resources.zip
 licenses-resources.zip
 ..\..\UXManager\Library\library-uxmanager-resources.zip
 ..\..\UXManager\Extensions\Apps\About\apps-about-resources.zip
+..\..\UXManager\Extensions\Apps\BlockSettings\apps-blocksettings-resources.zip
```

Comment:
Is BlockSettings appropriate name for this? Right now it only applies to Section.